package.json

```diff
@@ -34,21 +34,21 @@
   ],
   "license": "MIT",
   "dependencies": {
-    "elliptic-curve-solidity": "^0.2.1"
+    "elliptic-curve-solidity": "mariocao/elliptic-curve-solidity#chore/versions"
```

Comment:
I guess you will changes this now that the PR has been merged :)

Reply:
indeed :)